apps/api/src/domain/models/bibliographic-reference/entities/journalArticle-bibliographic-reference.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { DTO } from '../../../../types/DTO';
```

Comment:
rename this file `journal-article-bibliographic-reference.entity.ts`

Reply:
Done